extensions/amp-bind/0.1/bind-validator.js

```diff
@@ -311,6 +311,14 @@ function createElementRules_() {
       'state': null,
       'is-layout-container': null,
     },
+    'AMP-RENDER': {
+      'src': {
+        'allowedProtocols': {
```

Comment:
What is the purpose of `allowedProtocols`? Do we need to add `amp-state:` and `amp-script:` in this list?

Reply:
I just used the same as`amp-list` 🙂

Reply:
its for the bind-validator which attempts to mimic amp validator but at runtime